Fix #312  `tar08_davslav82.dlg`

th3w1zard1 opened the pull request:
Set LISTENER to Player after Canderous speaks, when greeting the female slave in Davik's Estate

Fixes #312

DarthParametric commented:
Your assigned entries were wrong. The entry directly after the companion's interjection is the one that needs it. I also realised that there's a separate DLG for the male slaves. Never noticed before since I don't play a female PC. Corrected the former, added the latter.

th3w1zard1 commented:
> Your assigned entries were wrong. The entry directly after the companion's interjection is the one that needs it. I also realised that there's a separate DLG for the male slaves. Never noticed before since I don't play a female PC. Corrected the former, added the latter.

Weird, I swear I double-checked the GFF paths. Didn't know about the male slave either.
